examples/js/loaders/FBXLoader2.js

```diff
@@ -571,11 +571,10 @@
 
 							var faceVertexBuffer = [];
 							var polygonIndex = 0;
-							for ( var polygonVertexIndex = 0; polygonVertexIndex < indexBuffer.length; ++ polygonVertexIndex ) {
+							indexBuffer.forEach( function ( vertexIndex, polygonVertexIndex, indexBuffer ) {
```

Comment:
I'm all for following code styles to keep things consistent, but can I ask why this is a rule?  It's supported by every major browser, and it helps keep temporary loop variables contained within the loop.  I could honestly just write a function and then call that function inside the loop (which isn't against his coding style), but at that point what would we be gaining?